src/Avalonia.Controls/DragDrop/DragOperation.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+
+namespace Avalonia.Controls.DragDrop
+{
+    [Flags]
+    public enum DragOperation
```

Comment:
Might be worth to rename to [DragDropEffects](https://docs.microsoft.com/en-us/dotnet/api/system.windows.dragdropeffects?view=netframework-4.7.1)

Reply:
Done.